test/test_mps.py

```diff
@@ -3543,6 +3547,116 @@ def test_slice(self):
         mps_slice4 = mps_x[1, :].to('cpu')
         self.assertEqual(cpu_slice4, mps_slice4)
 
+    def test_slice_reshape_view_api_test_1(self):
```

Comment:
These tests seems to have a lot of things in common.
1) are they not covered by OpInfo based testing already?
2) If not, should they be refactored such that they have a lot less boiler plate and consistent output metadata checking?
3) While making them generic, you can use parametrize decorator to make them run on all dtypes (avoiding int-specific tests)

Reply:
These tests are needed in addition to the opInfo tests, I'll parametrize them as you reccomended

Reply:
I've updated the tests, but I can't get parametrize to work. It keeps returning a missing argument error